<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Widget</class>
 <widget class="QWidget" name="Widget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>607</width>
    <height>515</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Widget</string>
  </property>
  <widget class="QWidget" name="gridLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>70</y>
     <width>561</width>
     <height>381</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="3" column="2">
     <widget class="QPushButton" name="pushButton_BorrarElemento">
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="text">
       <string>Eliminar
Elemento</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0" colspan="3">
     <widget class="QLabel" name="Resultado">
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>12</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>...</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="2" column="0" colspan="3">
     <widget class="QLineEdit" name="Numero"/>
    </item>
    <item row="0" column="0" colspan="3">
     <widget class="QLabel" name="Consigna_2">
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>18</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Programa de creación y 
modificación de Deque</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="3" column="1">
     <widget class="QPushButton" name="pushButton_AddFinal">
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="text">
       <string>Agregar al 
Final</string>
      </property>
     </widget>
    </item>
    <item row="3" column="0">
     <widget class="QPushButton" name="pushButton_AddFrente">
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="text">
       <string>Agregar al 
 Frente</string>
      </property>
     </widget>
    </item>
    <item row="1" column="0" colspan="3">
     <widget class="QLabel" name="Consigna">
      <property name="font">
       <font>
        <family>Arial</family>
        <pointsize>12</pointsize>
        <bold>true</bold>
       </font>
      </property>
      <property name="text">
       <string>Ingresar un número</string>
      </property>
      <property name="alignment">
       <set>Qt::AlignmentFlag::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="4" column="0">
     <widget class="QPushButton" name="pushButton_MostrarDeque">
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="text">
       <string>Mostrar 
Deque</string>
      </property>
     </widget>
    </item>
    <item row="4" column="2">
     <widget class="QPushButton" name="pushButton_BuscarElemento">
      <property name="cursor">
       <cursorShape>PointingHandCursor</cursorShape>
      </property>
      <property name="text">
       <string>Buscar 
Elemento</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
